packages/core/src/PlayerContextProvider.js

```diff
@@ -56,6 +56,8 @@ const defaultState = {
    * complete
    */
   awaitingResumeOnSeekComplete: false,
+  // true if media was playing when the active track changed
+  awaitingPlayAfterNextUpdate: false,
```

Comment:
This name is a bit confusing because "update" makes me think you're referring to a React component update, but I think you mean once the track has loaded.

Maybe `awaitingPlayAfterTrackLoad` would be good?

Reply:
changed